src/components/Tabs/Bar.js

```diff
@@ -58,6 +58,7 @@ const Bar = ({ children, className }) => {
       css={css`
         display: flex;
         width: 100%;
+        border-bottom: 1px solid var(--color-neutrals-300);
```

Comment:
This color is very bright in dark mode, maybe we could use something a little closer to the left rail separator.
<img width="731" alt="Screen Shot 2021-07-08 at 3 59 28 PM" src="https://user-images.githubusercontent.com/70179303/124983260-95986300-dfec-11eb-97a2-c74bacb3f3df.png">